client/packages/host/src/api/hooks/settings/namePropertyData.ts

```diff
@@ -28,23 +28,109 @@ export type LocalisedNamePropertyConfig = Partial<
 };
 
-const populationServed: ConfigureNamePropertyInput = {
-  id: '7716cecc-7d62-4f1b-93fa-a55a275079b4',
-  propertyId: POPULATION_SERVED_KEY,
-  key: POPULATION_SERVED_KEY,
-  name: 'Population Served',
-  valueType: PropertyNodeValueType.Float,
-  allowedValues: null,
-  remoteEditable: true,
+// English keys are fall-backs, all other languages should implement the same keys
+type TranslationKey = keyof typeof frTranslations;
+
+const enTranslations = {
+  POPULATION_SERVED_KEY: 'Population Served',
+  LATITUDE_KEY: 'Latitude',
+  LONGITUDE_KEY: 'Longitude',
+  SUPPLY_LEVEL_KEY: 'Supply Level',
+  FACILITY_TYPE_KEY: 'Facility Type',
+  OWNERSHIP_TYPE_KEY: 'Ownership Type',
+  BUFFER_STOCK_KEY: 'Stock Safety Buffer (months)',
+  SUPPLY_INTERVAL_KEY: 'Supply Interval (Months between deliveries)',
+  PACKAGING_LEVEL_KEY: 'Packaging Level',
+  ELECTRICITY_AVAILABILITY_KEY: 'Electricity Availability',
+  SOLAR_AVAILABILITY_KEY: 'Solar Availability',
+  GAS_AVAILABILITY_KEY: 'Gas Availability',
+  KEROSENE_AVAILABILITY_KEY: 'Kerosene Availability',
+  PENTA_3_KEY: 'Penta-3 Coverage',
+  ZERO_DOSE_KEY: 'Zero Dose Coverage',
 };
 
-export const gapsNameProperties: LocalisedNamePropertyConfig = {
-  en: [
+// French Translations
+const frTranslations = {
+  POPULATION_SERVED_KEY: 'Population Desservie',
+  LATITUDE_KEY: 'Latitude',
+  LONGITUDE_KEY: 'Longitude',
+  SUPPLY_LEVEL_KEY: "Niveau d'approvisionnement",
+  FACILITY_TYPE_KEY: "Type d'etablissement",
+  OWNERSHIP_TYPE_KEY: 'Type de propriété',
+  BUFFER_STOCK_KEY: 'Marge de sécurité des stocks (mois)',
+  SUPPLY_INTERVAL_KEY:
+    "Intervalle d'approvisionnement (Mois entre les livraisons)",
+  PACKAGING_LEVEL_KEY: 'Niveau de conditionnement',
+  ELECTRICITY_AVAILABILITY_KEY: 'Disponibilité de l’électricité',
+  SOLAR_AVAILABILITY_KEY: 'Disponibilité de l’énergie solaire',
+  GAS_AVAILABILITY_KEY: 'Disponibilité du gaz',
+  KEROSENE_AVAILABILITY_KEY: 'Disponibilité du kérosène',
+  PENTA_3_KEY: 'Couverture Penta-3',
+  ZERO_DOSE_KEY: 'Couverture Zéro Dose',
+  // Allowed values translations
+  Primary: 'Niveau national',
+  'Sub-National': 'Niveau régional',
+  'Lowest Distribution': 'Niveau district',
+  'Service Point': 'Point de service',
+  'National Vaccine Store': 'Dépôt national de vaccins',
+  'Regional Vaccine Store': 'Dépôt régional de vaccins',
+  'Referral Hospital': 'Hôpital Général de référence',
+  'Municipal Warehouse': 'Entrepôt municipal',
+  'Maternal Clinic': 'Clinique de maternité',
+  Government: 'Gouvernement',
+  NGO: 'ONG',
+  Private: 'Privé',
+  'Faith-based': 'Confessionnel',
+  'Primary (1)': 'Primaire (1)',
+  'Secondary (2)': 'Secondaire (2)',
+  'Tertiary (3)': 'Tertiaire (3)',
+  '> 16 hours': '> 16 heures',
+  '8-16 hours': '8-16 heures',
+  '< 8 hours': '< 8 heures',
+  'No availability': 'Aucun(e) disponibilité',
+  Unknown: 'Inconnu(e)',
+  Available: 'Disponible',
+  Irregular: 'Irrégulier',
+  High: 'Élevé(e)',
+  Medium: 'Moyen(ne)',
+  Low: 'Faible',
```

Comment:
Were these values confirmed by translators? looking at it now, wondering if we should pass in `t()`, so these can exist in weblate, be translated for other languages, or with custom translations when those are in, without dev input?

Up to you, maybe okay for now, more pressing work to do, maybe saves us a context switch in future 🤷‍♀️